src/Theme/Global.elm

```diff
@@ -118,23 +118,26 @@ globalStyles =
         , typeSelector "h1"
             [ fontFamilies [ "Adelle", "serif" ]
             , color purple
-            , Theme.FluidScale.fontSize4
+            , Theme.FluidScale.fontSize1
```

Comment:
Was it from small to large before? Because oddly, that makes more sense to me. Mostly typography scale go like:
smaller than base
base
medium
large
extra large.

They should not necessarily correspond with heading text because they need to be used with ALL text.

Having said that, please don't undo your work here. It clearly makes sense on some level or you would not have done it! Warning that it may cause conflicts that could go unseen. So once this tranche is all merged in have a scan through all the content type pages to eye-ball as intended.

Reply:
this looks resolved to me?